Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift

```diff
@@ -106,4 +106,18 @@ class MSDistributeViewController: UITableViewController, AppCenterProtocol {
   @IBAction func customizedSwitchUpdated(_ sender: UISwitch) {
     UserDefaults.standard.set(sender.isOn, forKey: kSASCustomizedUpdateAlertKey)
   }
+    
+  func exitWithSimulateUpdate() {
+    let UPDATE_SIMUATION_TIMEOUT = 7.0
+    print("distributeWillExitApp callback invoked");
+    let alertController = UIAlertController(title: NSLocalizedString("distribute_alert_willExit_title", tableName: "Sasquatch", comment: ""),
+            message: NSLocalizedString("distribute_alert_willExit_message", tableName: "Sasquatch", comment: ""),
+            preferredStyle: .alert)
+
+    // Show alert saying that the app is closing
+    self.present(alertController, animated: true)
+    DispatchQueue.main.asyncAfter(deadline: .now() + UPDATE_SIMUATION_TIMEOUT) {
+      exit(0)
```

Comment:
This is not correct. We should use Distribute API, otherwise this simulation is useless.

Something like showing an alert, then waiting 7 sec and then calling `appCenter.closeApp()` should do the job.

Reply:
Actually, it's also not the right way to simulate this.
The correct way would be creating a new distribute listener, or add a flag to AppDelegate.
Alternatively we can delete this simulation. It only exists in ios demo app anyways.
Let's discuss it offline.